kurtosis/src/networks/kurtosis-devnet/network-configs/genesis.json

```diff
@@ -31,7 +31,7 @@
   "timestamp": "0x0",
   "alloc": {
     "0x20f33ce90a13a4b5e7697e3544c3083b8f8a51d4": {
```

Comment:
Had to increase the balance as there was insufficient balance as per a check in PoL SC.